lib/index.js

```diff
@@ -9,7 +9,8 @@ module.exports = function (startFn, _options) {
       port: options.port || 8080,
       debug: options.debug || false,
       hardShutdownDelay: options.hardShutdownDelay || 60 * 1000,
-      env: options.env || function (index) { return { stickycluster_worker_index: index }; }
+      env: options.env || function (index) { return { stickycluster_worker_index: index }; },
+      errorHandler: options.errorHandler || function (err) { console.log(err); process.exit(1); }      
```

Comment:
Is this what happens if we were to omit a handler altogether? I was thinking that our fallback would be a `no-op` function.

Reply:
Hi @gboysko , 

Yes, this happens when we omit to specify a handler. I can change the default handler to be a `no-op` function, but I think it's better to log the error details and exit with an error code. If we do nothing the process would be hanged and never exit.